ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -61,42 +61,12 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
     assert.dom(overview.createDestination).hasText('Create new destination', 'Toolbar action renders');
   });
 
-  module('community', function (hooks) {
```

Comment:
should we have a test for this situation:

> Displays only one upsell badge for HVD managed clusters. We cannot tell which tier they are on. In the future this will likely change.